include/editor.hpp

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+#include "constants.hpp"
+#include "types.hpp"
+
+#include <cstddef>
+#include <fstream>
+#include <nlohmann/json.hpp>
+
+namespace vpd
+{
+/* @brief Class for vpd write
+ *
+ */
+class vpdEditor
+{
+  public:
+    vpdEditor() = delete;
+    vpdEditor(const vpdEditor&) = delete;
+    vpdEditor& operator=(const vpdEditor&) = delete;
+    vpdEditor(vpdEditor&&) = delete;
+    vpdEditor& operator=(vpdEditor&&) = delete;
+    ~vpdEditor()
+    {
+    }
+
+    /* @brief Construct vpdEditor class
+     *
+     * param[in] path - vpd file path
+     * param[in] inventoryPath - objectPath
+     * param[in] data - new data to write
+     * param[in] keyword - keyword going to write
+     * param[in] record - record goingt to write
+     */
+
+    vpdEditor(const std::string& path,
+              const sdbusplus::message::object_path& inventoryPath,
```

Comment:
DO we really need both file path and inventory path, from user?
As we have json and if file path provided, we can get the inventory path.
Others inputs please .

Reply:
That is what you need to come up with and see if we can replace that.

Reply:
ok, will handle in next patch/ additional commit